cli/ballerina-cli/src/main/java/io/ballerina/cli/task/CompileTask.java

```diff
@@ -42,20 +44,23 @@
 import org.wso2.ballerinalang.util.RepoUtils;
 
 import java.io.PrintStream;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Don't use '*' in importing. Import only the necessary ones 
```suggestion
import java.util.*;
```